docs/contributing/03-stories/story-04.md

```diff
@@ -2,17 +2,17 @@
 
 > **Status**: Done
 
-This is the story of Monada Sprint 4. It is told from the perspective of Mona
```

Comment:
We traditionally don't change old stories. No rewriting of history :-) Please revert.